### ISSUE TITLE
SAML as a signInProvider  is not supported in version FirebaseUI 6.0.0

### ISSUE THREAD

**Issue by shivna-2020** [firebase/firebaseui-web#898]
### Environment

  * Operating System version:  Windows 10
  * Browser version: Chrome Version 95.0.4638.69
  * Firebase UI version: 6.0.0
  * Firebase SDK version: 9.2.0

### The problem
Login with SAML does not work. It was working with Firebase UI version 5.0.0 and Firebase SDK 8.10.0

#### Steps to reproduce:

Setup up SAML using GCIP as follows:
```
signInOptions: [{
                provider: 'saml.auth0',
                providerName: 'SAML',
                buttonColor: '#2F2F2F',
                iconUrl:'https://www.gstatic.com/firebasejs/ui/2.0.0/images/auth/mail.svg'
}]
```

#### Relevant Code:

The following error is shown in the console.
```
Uncaught TypeError: firebase_compat_app__WEBPACK_IMPORTED_MODULE_0__.default.auth.SAMLAuthProvider is not a constructor
    at Ql (esm.js:281)
    at Rl (esm.js:284)
    at esm.js:341
    at b (esm.js:340)
    at esm.js:7
    at xn.<anonymous> (esm.js:197)
    at ye (esm.js:70)
    at xe (esm.js:69)
    at dj (esm.js:178)
    at cj.push.../node_modules/firebaseui/dist/esm.js.cj.h (esm.js:177)
```
```

**Comment by xil222** [comment]
@shivna-2020 
Thanks for the post, I think its an issue with the Firebase compat library doesn't support this library.
I'll file an internal bug.

**Comment by xil222** [comment]
should be a 9.2.1 version fixing it by the end of this week.

**Comment by xil222** [comment]
should be fixed in Firebase JS SDK [v9.3.0](https://firebase.google.com/support/release-notes/js#authentication)

**Comment by shivna-2020** [comment]
Tested and confirming that this has been fixed in v9.3.0.

**Comment by xil222** [comment]
cool
